src/6model/serialization.c

```diff
@@ -10,7 +10,7 @@
 
 /* Version of the serialization format that we are currently at and lowest
  * version we support. */
-#define CURRENT_VERSION 19
+#define CURRENT_VERSION 20
```

Comment:
Yes, but only need when you added version check code. like this: https://github.com/pmurias/MoarVM/blob/ae8d3838635a8443f1fba0e95b2ade541cd6902c/src/6model/serialization.c#L1657